runtime/onert/backend/train/TensorManager.cc

```diff
@@ -90,7 +90,7 @@ void TensorManager::claimNonConstPlan(const ir::OperandIndex &index)
   auto tensor = _tensors->getNonConstTensor(index);
   assert(tensor && !tensor->is_dynamic());
 
-  auto size = tensor->total_size();
+  auto size = ((tensor->total_size() - 1) | (_align - 1)) + 1;
```

Comment:
@jyoungyun :+1:
Could you please introduce macro like `aligned_size(size, alignment)`?